lua/kulala/ui/init.lua

```diff
@@ -127,6 +127,7 @@ local function open_kulala_window(buf)
     win_config = { split = config.split_direction == "vertical" and "right" or "below", win = request_win }
   end
 
+  vim.tbl_extend("force", win_config, config.ui.win_opts or {})
```

Comment:
```lua
win_config = vim.tbl_extend("force", win_config, config.ui.win_opts or {})
```
I think we have assign the returned value here

Reply:
You are absolutely right! Good catch!